node/cn/filters/api.go

```diff
@@ -69,6 +69,9 @@ type PublicFilterAPI struct {
 	events    *EventSystem
 	filtersMu sync.Mutex
 	filters   map[rpc.ID]*filter
+
+	// this field is for test. it makes the filter timeout more flexible when testing
+	timeout time.Duration
```

Comment:
Actually, this field need not be test-only. geth 1.11.5 has this field https://github.com/ethereum/go-ethereum/blob/v1.11.5/eth/filters/api.go.

Reply:
Yeah, klaytn haven't brought the change you mentioned, so the explanation is appropriate for now :)